### DISCUSSION TITLE
App internationalization issue

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#9775] [General - Components / Directives / etc] [answered]
I have followed the steps described on this page https://github.com/quasarframework/quasar/discussions/new , however I am receiving the following warnings on running the project with 'quasar dev':

```
"export 'Text' was not found in 'vue'
 warning  in ./node_modules/vue-i18n/dist/vue-i18n.esm-bundler.js
"export 'Text' was not found in 'vue'
 warning  in ./node_modules/vue-i18n/dist/vue-i18n.esm-bundler.js
"export 'computed' was not found in 'vue'
 warning  in ./node_modules/vue-i18n/dist/vue-i18n.esm-bundler.js
```

...and this in the console:

```
Uncaught TypeError: Object(...) is not a function
    at createComposer (webpack-internal:///./node_modules/vue-i18n/dist/vue-i18n.esm-bundler.js:177)
```

**Comment by i-mage** [comment]
So, I got rid of the warnings by changing line in i18n.js:

`import { createI18n } from 'vue-i18n' -> import { createI18n } from 'vue-i18n/index'`

But now I have another error in the console:
`vue.ref is not a function`

**Comment by hawkeye64** [comment, marked as answer]
Sounds like it could be a mismatch of wrong versions. Are you using Quasar v2 (which uses Vue 3)?
If so, then I think you are looking for this one: https://github.com/intlify/vue-i18n-next

**Reply by i-mage** [reply]
@quasar/cli: 1.1.3 → 1.2.1
@quasar/extras: 1.9.8 → 1.10.7
quasar: 1.14.1 → 1.15.21
@quasar/app: 2.1.1 → 2.2.10
@quasar/quasar-app-extension-qenv: 1.0.3 → 1.1.0

Not sure whether it is 1.14.1 (quasar) or 2.1.1 (@quasar/app)

**Reply by hawkeye64** [reply]
You're using Quasar v1, which uses Vue v2
This error, from above `vue.ref is not a function` implies Vue v3 is needed
In which case, you should upgrade to Quasar v2 or downgrade your version of vue-i18n

**Reply by i-mage** [reply]
Thanks for the info! :-) The support here is amazing.

**Reply by i-mage** [reply]
Meanwhile I managed to implement it into the project reading this article:

https://medium.com/@nafaabout/app-internationalization-with-quasar-framework-bfc222e6247c
